lib/rules/jest/no-if.js

```diff
@@ -0,0 +1,93 @@
+const {docsUrl} = require('../../utilities');
+
+const TEST_FUNCTION_NAMES = [
```

Comment:
We don't have this list for any other test?

Reply:
moved to a require from `utilities/jest`